internal/conns/awsclient.go

```diff
@@ -58,6 +58,10 @@ func (client *AWSClient) RegionalHostname(prefix string) string {
 	return fmt.Sprintf("%s.%s.%s", prefix, client.Region, client.DNSSuffix)
 }
 
+// ****************
+// TODO: REVIEW
+// TODO: AWS SDK for Go v2 does NO URL cleaning.
+// ****************
```

Comment:
Leftover comments?

Reply:
See above.

Reply:
I will leave this one in until the remaining uses in S3 Bucket are replaced.